integration_tests/src/main/python/hive_parquet_write_test.py

```diff
@@ -28,6 +28,9 @@
 _hive_bucket_gens = [
     boolean_gen, byte_gen, short_gen, int_gen, long_gen, string_gen, float_gen, double_gen,
     DateGen(start=date(1590, 1, 1)), _restricted_timestamp()]
+_hive_bucket_gens_sans_bools = [
+    byte_gen, short_gen, int_gen, long_gen, string_gen, float_gen, double_gen,
+    DateGen(start=date(1590, 1, 1)), _restricted_timestamp()]
```

Comment:
Nit: easier to read and maintain if we build one in terms of the other.
```suggestion
_hive_bucket_gens_sans_bools = [
    byte_gen, short_gen, int_gen, long_gen, string_gen, float_gen, double_gen,
    DateGen(start=date(1590, 1, 1)), _restricted_timestamp()]
_hive_bucket_gens = [boolean_gen] + _hive_bucket_gens_sans_bools
```

Reply:
Addressing these now.